src/grandorgue/sound/GOSoundAudioSection.cpp

```diff
@@ -734,15 +733,15 @@ void GOAudioSection::Setup(
         (end_length - copy_len) * m_BytesPerSample);
       if (fade_len > 0)
         DoCrossfade(
-          end_seg.end_data,
-          MAX_READAHEAD,
-          (const unsigned char *)pcm_data,
-          start_seg.start_offset - fade_len,
-          pcm_data_channels,
-          m_BitsPerSample,
-          fade_len,
-          loop_length,
-          end_length);
+          end_seg.end_data,                  // dest
```

Comment:
When I discovered #1724 it was too difficult for me to understand how `DoCrossfade` worked and to trace wthat value were passed to each parameter because this method had lots of parameters. So I want to mark just the parameter names for making this job easier. So I'd leave them in comments

Reply:
If they should remain, they should explain that what you're trying to say now. For instance something like:

// dest is the name of the receiving parameter that is a pointer to the actual output sample data to use